package.json

```diff
@@ -51,5 +53,6 @@
     "ts-jest": "^29.2.5",
     "ts-node": "^10.9.2",
     "typescript": "^5.5.4"
-  }
+  },
+  "packageManager": "pnpm@9.9.0+sha1.3edbe440f4e570aa8f049adbd06b9483d55cc2d2"
```

Comment:
[q] I am assuming this was not intended, and have removed it.

Reply:
No this was not intended. I am using corepack to manage pnpm and it keeps adding this field automatically everytime I execute some pnpm command. No idea how to remove it.

```txt
! The local project doesn't define a 'packageManager' field. Corepack will now add one referencing pnpm@9.9.0+sha1.3edbe440f4e570aa8f049adbd06b9483d55cc2d2.
! For more details about this field, consult the documentation at https://nodejs.org/api/packages.html#packagemanager
```